loopchain/channel/channel_service.py

```diff
@@ -472,7 +504,7 @@ def connect_to_radio_station(self, is_reconnect=False):
                 peer_object=b'',
                 peer_id=ChannelProperty().peer_id,
                 peer_target=ChannelProperty().peer_target,
-                group_id=ChannelProperty().group_id),
+                group_id=ChannelProperty().peer_id),
```

Comment:
This group_id should be deleted.

Reply:
`group_id` is partially removed here. Group ids related to grpc proto will be updated at the next pull request.